### PR TITLE
Review branch

### PR DESCRIPTION
**Pull request by peterdgreat**
This is a simple "To-do list" which consists of four basic operations; Create, Read, Update, Delete (CRUD) This was built using vanilla JavaScript and Webpack

- Linters setup

-webpack setup

- class created for basic operations

- functions to add, delete, delete all created and logic implemented

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+- Javascript
+
+- Visual Studio Code
+
+- Git & Github
+
+
+## Live Server
+[Todo- List](https://hopeful-brown-dda6d4.netlify.app/)
```
Comment (DyaryRaoof) [inline README.md]: your live demo doesn't work please try and fix that.

**Review by DyaryRaoof — CHANGES_REQUESTED** [review]
### Changes Requested ♻️ 

Hi. Great work so far. 💪  the site is functional however there are a few things you can work on: 



- There are no Linters in your PR please take care of that.
-  I found a few bugs in your code. when you click on a task it highlights it as if you have pressed the detail ( three dots icon), but I think you should only highlight the task if the detail icon is pressed.
- second, if you click on two items and deselect them by clicking on the edge you will see the value of both tasks ( the description of each) become the same.
- I would like to say that your code is very efficient. It's aligned with JavaScript best practices, HTML and CSS best practices, and it is aligned with DRY, KISS, and YAGNI principles of programming. So good job 👍

File: README.md (hunk 1 above)
Comment (peterdgreat) [reply README.md]: Thanks

Comment (DyaryRaoof) [reply README.md]: Welcome, Peter.

Comment (peterdgreat) [reply README.md]: Can you check the live view here please https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/
The edit feature on the li item and not the icon

Comment (DyaryRaoof) [reply README.md]: Oh. Yes. You are totally right. please forgive the mistakeke

**Review by DyaryRaoof — APPROVED** [review]

**Pull request merged**